dojo/models.py

```diff
@@ -1122,8 +1122,8 @@ class Finding(models.Model):
     title = models.TextField(max_length=1000)
     date = models.DateField(default=get_current_date)
     cwe = models.IntegerField(default=0, null=True, blank=True)
-    cve_regex = RegexValidator(regex=r'^CVE-\d{4}-\d{4,7}$',
-                                 message="CVE must be entered in the format: 'CVE-9999-9999'. ")
+    cve_regex = RegexValidator(regex=r'^[A-Z]{1,3}-\d{4}-\d{4,7}$',
+                                 message="Vulnerability ID (CVE, WS. BlacDuck) must be entered in the format: 'ABC-9999-9999'. ")
```

Comment:
I would remove the references in parentheses.

Reply:
Ok, so it will be look like this